src/catkin_pkg/cli/generate_changelog.py

```diff
@@ -43,6 +43,9 @@ def main(sysargs=None):
     parser.add_argument(
         '-a', '--all', action='store_true', default=False,
         help='Generate changelog for all versions instead of only the forthcoming one (only supported when no changelog file exists yet)')
+    parser.add_argument(
+        '--only-merges', action='store_true', default=False,
+        help='Skip all commits but merge commits to the changelog')
```

Comment:
Please make this new argument mutually exclusive with the existing argument `--skip-merges` since both shoudn't be passable at the same time.

Reply:
Makes sense. Added a change, and tested:
```
$ catkin_generate_changelog --only-merges --skip-merges
Found packages: pkg_1, pkg_1_support
Querying commit information since latest tag...
ERROR: Both "skip_merges" and "only_merges" are set to True, which contradicts.
```
```
$ catkin_generate_changelog --only-merges 
Found packages: pkg_1, pkg_1_support
Querying commit information since latest tag...
Updating forthcoming section of changelog files...
- updating 'pkg_1/CHANGELOG.rst'
- updating 'pkg_1_support/CHANGELOG.rst'
Done.
```

Reply:
Please remove the custom logic and use [mutually exclusive group](https://docs.python.org/3/library/argparse.html#mutual-exclusion).